Makefile

```diff
@@ -62,15 +62,26 @@ clean:
 	@find . -name "__pycache__" -type d -exec rm -rf {} +
 
 # Mintlify commands (run from build directory where final docs are generated)
-mint-broken-links:
-	@echo "Checking for broken links..."
-	@cd build && mint broken-links
+# Note: mint must be installed globally via npm
+mint-broken-links: build
```

Comment:
This is the make command that should be run from the script above.

Not many people can read `awk` these days, so it's hard to maintain. 

I'd suggest to do something like this the following:

1. move this into either a shell script or a python script
2. in the script, capture the output from mint broken-links
3. then try to come up with an easy to read / easy to understand way to validate all the links


Process should look like:
1. run broken links
2. something yields individual links
3. next entity validates each link